requirements.txt

```diff
@@ -1,6 +1,10 @@
-python_version>=3.11
+#python_version>=3.11
 aiohttp
 beautifulsoup4
 cryptography
 lxml
 pyjwt
+
+
+pandas
```

Comment:
I would really prefer not to depend on pandas for just one export function. Pandas is a big framework to depend on. Can we do this differently?

Reply:
Absolutely.
I can simplify that, as well as I can add multiple checks around the code (e.g. validation of folder path etc).

Note: The first thing is - I wanted to check if you were fine with considering this as a relevant addition to your codebase. After that, technical aspects are straightforward.